flask_dance/contrib/azure.py

```diff
@@ -76,6 +79,15 @@ def make_azure_blueprint(
     :returns: A :doc:`blueprint <flask:blueprints>` to attach to your Flask app.
     """
     scope = scope or ["openid", "email", "profile", "User.Read"]
+    token_url = f"https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token"
+    authorization_url = (
+        f"https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize"
+    )
+    auto_refresh_url = None  # see #161
```

Comment:
What does `see #161` mean?

Reply:
It's a reference to the issue where the "idea" of setting `auto_refresh_url ` to `None`, if the `offline` option is set to `False`. I guess it might be unnecessary - I can just remove it?

https://github.com/singingwolfboy/flask-dance/pull/161

Reply:
Yeah, I think it can be removed.